ethstorage/storage_manager_test.go

```diff
@@ -12,99 +14,98 @@ import (
 	"github.com/ethstorage/go-ethstorage/ethstorage/eth"
 )
 
-
```

Comment:
The same as the sync test file

Reply:
This is the automatic formatting of the code